azext_edge/edge/providers/check/mq.py

```diff
@@ -576,11 +262,11 @@ def evaluate_brokers(
         check_manager.add_display(target_name=target_brokers, namespace=namespace, display=Padding(brokers_count_text, (0, 0, 0, 8)))
 
         added_distributed_conditions = False
+        added_diagnostics_conditions = False
```

Comment:
I see double

Reply:
can you elaborate on the comment please? I don't understand😳

Reply:
I cannot read